storefront/app/controllers/spree/checkout_controller.rb

```diff
@@ -152,14 +152,6 @@ def remove_store_credit
 
     private
 
-    def store_user_location
```

Comment:
We should always store the `request.fullpath` because we need to redirect users to the checkout when they log in